### ISSUE TITLE
`Datagrid` unable to input text in `DialogHost`

### ISSUE THREAD

**Issue by Coloryr** [AvaloniaUtils/DialogHost.Avalonia#47]
I put a datagrid in dialoghost, but it can be edit
```
<dialog:DialogHost DisableOpeningAnimation="True" Identifier="Main">
    <dialog:DialogHost.DataTemplates>
        <DataTemplate DataType="vm:AddAskModel">

            ....

            <DataGrid
                Width="200"
                Height="150"
                Margin="5,0,0,0"
                AutoGenerateColumns="False"
                CanUserResizeColumns="True"
                CanUserSortColumns="False"
                ItemsSource="{Binding ServerList}">
                <DataGrid.Columns>
                    <DataGridTextColumn
                        Binding="{Binding Name}"
                        Header="执行的服务器"
                        IsReadOnly="False" />
                </DataGrid.Columns>
            </DataGrid>
        </DataTemplate>
    </dialog:DialogHost.DataTemplates>

....

```

![GIF](https://github.com/AvaloniaUtils/DialogHost.Avalonia/assets/26276037/7b0df6bb-b679-4d8f-b471-041117a024fb)

To be precise, it is impossible to input text after selecting it

**Comment by SKProCH** [comment]
Hello, @Coloryr 

To be honest, I didn't understand what the problem was. Can you rephrase it? It might be worth specifying the expected behavior and the actual behavior also.

**Comment by Coloryr** [comment]
It may be a problem with the Chinese input method. When I input English, I can input it all at once, but inputting Chinese through Pinyin is not possible

**Comment by SKProCH** [comment]
That affects only DataGrid inside Dialogs?

**Comment by Coloryr** [comment]
https://github.com/AvaloniaUtils/DialogHost.Avalonia/assets/26276037/96235da2-868e-4818-8430-49d9f21aec0d

**Comment by SKProCH** [comment]
@Coloryr is this issue still relevant? (Sorry, i didn't have time to check, my bad)

**Comment by Coloryr** [comment]
> @Coloryr is this issue still relevant? (Sorry, i didn't have time to check, my bad)

No test, wait a second.

**Comment by Coloryr** [comment]
It can be work now.
